requirements-base-runtime.txt

```diff
@@ -6,7 +6,6 @@
 #
 aiofiles==23.2.1
     # via quart
-awscli==1.32.39
```

Comment:
@alexrashed The AWS CLI got introduced here by #10217, `awscli` is not a dependency mentioned in `base-runtime` in `setup.cfg`. What will we do here so that the dependency and asf upgrades do not revert each others changes here?

Reply:
Great catch! It's definitely (1), and was a mistake when initially introducing the pin upgrade together with the ASF updates in #10183. I added a quick commit in this PR to avoid this issue in the future.